### PR TITLE
[Charts/Portal] Upgrading RMQ chart to simplify the volume permissions conf

### PR DESCRIPTION
**Pull request by ksaladi**
**Description of the change**

- Upgrading the RMQ chart version from 7.6.6 to 12.0.3 to simplify the volumePermissions so that customer can configure the fsGroup and runAsUser values other than non-root, this requirement came from Adobe where they want to configure/mount the volume other than root.

**Tests**

- Installed Portal and verified the on-demand and auto proxies work as it is working without this change.

**Benefits**

**Drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[charts/gateway]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

**Comment by VodelaPremChander** [comment]
Have the installation tested using the properties you have added - 
with helm install example "helm install --set rabbitmq.volumePermissions.enabled=true"

**Comment by ksaladi** [comment]
>  yes tested @premchandervodela

File: charts/portal/README.md [hunk 1]
```diff
@@ -461,6 +461,9 @@ this feature.
 | `rabbitmq.serviceAccount.create`| Enable creation of ServiceAccount for Bitnami RabbitMQ |`true`|
 | `rabbitmq.serviceAccount.name`| Name of the created serviceAccount | Generated using the `portal.fullname` template |
 | `rabbitmq.rbac.create`| Create & use RBAC resources |`true`|
+| `rabbitmq.volumePermissions.enabled` | Enable init container that changes the owner and group of the persistent volume(s) mountpoint to `runAsUser:fsGroup`. After enabling set the appropriate runAsUser and fsGroup values |`false`|
+| `rabbitmq.podSecurityContext.fsGroup` | Set RabbitMQ pod's Security Context fsGroup | `1001` |
```
Comment (as666499) [inline charts/portal/README.md:465]: Do we have to set podSecurityContext.enabled to true if we have to pass custom value for rabbitmq.podSecurityContext.fsGroup?

File: charts/portal/README.md [hunk 2]
```diff
@@ -461,6 +461,9 @@ this feature.
 | `rabbitmq.serviceAccount.create`| Enable creation of ServiceAccount for Bitnami RabbitMQ |`true`|
 | `rabbitmq.serviceAccount.name`| Name of the created serviceAccount | Generated using the `portal.fullname` template |
 | `rabbitmq.rbac.create`| Create & use RBAC resources |`true`|
+| `rabbitmq.volumePermissions.enabled` | Enable init container that changes the owner and group of the persistent volume(s) mountpoint to `runAsUser:fsGroup`. After enabling set the appropriate runAsUser and fsGroup values |`false`|
```
Comment (as666499) [inline charts/portal/README.md:464]: Can we document persistent parameters added in [12.0.3](https://github.com/bitnami/charts/tree/2858f965ea5503bd91006fd01de59562260c4cb6/bitnami/rabbitmq) which were not there in [7.6.6](https://github.com/bitnami/charts/tree/1b9ca8daea338aed3949b1857a94c8a5ac451fdc/bitnami/rabbitmq)?

Comment (ksaladi) [reply charts/portal/README.md:464]: @as666499 Only 'mountPath' is the difference. Added that part now.

File: charts/portal/README.md (hunk 1 above)
Comment (ksaladi) [reply charts/portal/README.md:465]: podSecurityContext .enabled is true by default

**Review by as666499 — APPROVED** [review]

**Review by APIM-KiranVaddadi — APPROVED** [review]

**Pull request merged**